pydantic_ai_slim/pydantic_ai/models/openai/responses.py

```diff
@@ -1446,24 +453,24 @@ async def _responses_create(  # noqa: C901
                 input=openai_messages,
                 model=self.model_name,
                 instructions=instructions,
-                parallel_tool_calls=model_settings.get('parallel_tool_calls', OMIT),
-                tools=tools or OMIT,
-                tool_choice=tool_choice or OMIT,
-                max_output_tokens=model_settings.get('max_tokens', OMIT),
+                parallel_tool_calls=model_settings.get('parallel_tool_calls', omit),
```

Comment:
Why are those being changed?

Reply:
yeahp that was unnecessary, I moved the import to `_shared.py` and put this one and `chat.py` back to how it was before